### DISCUSSION TITLE
Getting back gesture to work on iOS

### DISCUSSION THREAD

**Discussion by graemian** [apache/cordova#447] [Q&A] [answered]
Hi,

The back gesture works as expected (i.e. navigates to the previous page) when I run my Meteor Cordova app on Android, but it does nothing on iOS.

Can anyone advise on how to get it working on iOS?

Thanks!

**Comment by breautek** [comment]
By [default](https://developer.apple.com/documentation/webkit/wkwebview/1414995-allowsbackforwardnavigationgestu?language=objc) it seems to be disabled and I'm not seeing where Cordova may enable it, so I'd suspect that's why it's not working on iOS.

I think a PR would be welcome to introduce a config.xml preference flag. *Might* be possible to enable it via a plugin as well.

**Comment by graemian** [comment, marked as answer]
I found that you can enable it at runtime like this:

`window.WkWebView.allowsBackForwardNavigationGestures(true)`
